src/tiledimage.js

```diff
@@ -660,6 +667,8 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
         $.console.assert(!newClip || newClip instanceof $.Rect,
             "[TiledImage.setClip] newClip must be an OpenSeadragon.Rect or null");
 
+//TODO: should this._raiseBoundsChange(); be called?
```

Comment:
What about raising both?
OSD is using the `bounds-change` event in multiple places to compute the content bounds.
On the other hand, we have a separate `getClippedBounds` method so not having `bounds-change` raised when `getBounds` doesn't change makes sense.